GHA-324 PullRequestClosed: Add fix-version parameter (#372)
SonarSource/gh-action-lt-backlog: PullRequestClosed/ReadMe.md

This parameter will resolve only issues created by automation, or bots.

### `fix-version`

> Jira fix version name to assign when the pull request is merged, such as `10.23`. Leave empty to skip. The fix version is set only when the linked issue has none yet. Failures are logged but do not fail the workflow.

Reviewer: Simple version
```suggestion
Jira fix version name assigned to issues without fix version set. Example: `10.23`

This parameter is optional.
```

          github-token: ${{secrets.GITHUB_TOKEN}}
          jira-user:  ${{ fromJSON(steps.secrets.outputs.vault).JIRA_USER }}
          jira-token: ${{ fromJSON(steps.secrets.outputs.vault).JIRA_TOKEN }}

>           fix-version: '10.23'

Reviewer: Let's don't put this, so it's generally easy to copy/paste. We can add it in the next PR with the automatic value
```suggestion
```
